chatbot/app.js

```diff
@@ -33,6 +33,25 @@ const EP_IMAGES = 'https://s3-us-west-2.amazonaws.com/<link_to_images>/';
 
 let Cortex = require("./cortex");
 let urlExists = require('url-exists');
+let firebase = require('firebase-admin');
+require("firebase/firestore"); // required for "side-effets"
+
+let serviceAccount = require('<Service_Account>.json');
+
+firebase.initializeApp({
+  apiKey: '<API_Key>',
```

Comment:
Depending how many of these configurations we have (we also need to document them), we may consider moving them out to a separate config file. @CPinelli